app/assets/javascripts/admin/util.js.erb

```diff
@@ -1,6 +1,6 @@
 $(document).ready(function() {
   $('.datetimepicker').datetimepicker({
-    dateFormat: Spree.translations.date_picker,
+    dateFormat: "yy-mm-dd",
```

Comment:
Or did you mean change the default Spree setting as well as this? Maybe both would be preferable...?